model/operation.h

```diff
@@ -206,8 +206,27 @@ namespace minsky
   class DataOp: public ItemT<DataOp, Operation<minsky::OperationType::data>>
   {
     CLASSDESC_ACCESS(DataOp);
+    friend struct SchemaHelper;
+    string m_description;
   public:
-    string description;
+    DataOp() {description("");}
+    // ensure that copies create a new data operation
+    DataOp(const DataOp& x): 
+      OperationBase(x){group.reset(); description(x.description());}
```

Comment:
Whilst probably harmless, I was wondering why you added group.reset() here. group.reset is already done in ItemT::clone. None of the other item copy constructors have this, except for IntOp. For consistency, I suggest removing the group.reset here and in IntOp. You can log this as a low priority backlog ticket, however, if your'e concerned we might not understand the implications of that change.

Reply:
Sorry, I missed this and will commit the changes after the current Travis build has finished. It didn't occur to me that the group.reset() in ItemT:clone() would work for these objects too. I sometimes forget that these and other objects fall under the item object umbrella.